packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -214,6 +214,20 @@ abstract class WidgetsBindingObserver {
   ///    boilerplate.
   void didChangeTextScaleFactor() { }
 
+  /// Called when the platform's desired brightness changes.
+  ///
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
+  /// render in a "dark mode". That change is reflected in this property.
+  ///
+  /// Not all platforms necessarily support a concept of brightness mode. Those
+  /// platforms will report [Brightness.light] in this property.
+  /// 
+  /// See also:
+  /// 
+  ///  * [MediaQuery.of], which provides a similar service with less
+  ///    boilerplate.
+  void didChangePlatformBrightness() { }
```

Comment:
It should probably be documented above how one could find out about the new platform brightness after this fires.

Reply:
This information will be added in the next push.